.github/actions/check-merge-conflict/action.yml

```diff
@@ -7,7 +7,7 @@ runs:
     - name: Check for merge conflict
       if: github.event.pull_request == '' || github.event.pull_request.head.repo.fork == false
       # releases/2.x
-      uses: eps1lon/actions-label-merge-conflict@b8bf8341285ec9a4567d4318ba474fee998a6919
+      uses: eps1lon/actions-label-merge-conflict@v2.1.0
```

Comment:
https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows+and+actions#:~:text=For%20non%2DGitHub%20actions